athena-aws-cmdb/src/test/java/com/amazonaws/athena/connectors/aws/cmdb/AwsCmdbMetadataHandlerTest.java

```diff
@@ -154,7 +155,8 @@ public void doListSchemaNames()
     @Test
     public void doListTables()
     {
-        ListTablesRequest request = new ListTablesRequest(identity, queryId, catalog, "schema1");
+        ListTablesRequest request = new ListTablesRequest(identity, queryId, catalog, "schema1",
```

Comment:
might be ok to have a convenience constructor which uses a NULL next token to make initial requests easier.

Reply:
Your comment for me from before was to only have one constructor for ListTablesRequest and ListTablesResponse. This comment seems contradictory.

Reply:
previously you had two constructors - one with pagination and one without. I'm proposing a 2nd constructor that has a convince of 'null' for the NextToken but still requires the pageSize. This doesn't leave an avenue for ignoring pagination support like your constructors did. That is the difference here. Its a suggestion, not a requirement so won't hold back the PR. Just something to think about.